docs/modules/ROOT/pages/troubleshooting/index.adoc

```diff
@@ -118,7 +119,11 @@ If not, link it to the `appstudio-pipeline` service account with:
 ----
 secret_name=secret-for-my-registry
 
-kubectl secret link appstudio-pipeline "$secret_name"
+# using oc
+oc secrets link appstudio-pipeline "$secret_name"
+
+# using kubectl
+kubectl patch serviceaccount appstudio-pipeline -p '{"secrets": [{"name": "$secret_name"}]}'
```

Comment:
Do we need to be specific about `oc` usage here? I tried to describe everything in terms of `kubectl` within the docs.

Reply:
not really, but I was confused if that was the actual intent of the original author, since `kubectl secret link` doesn't exist, but `oc secrets link` does.

I added the correct kubectl version (which is a bit cumbersome being a patch op., specially compared to the `oc` version)

But I do agree it stands out a bit since it's the only `oc` command. I can remove it then, would that be better then ?

Reply:
Ah. Good catch. How about if you switch the two to put kubectl first then oc? The oc command is certainly simpler!

Reply:
done! 

I've put the kubectl first, then left the oc one 

I actually caught a "quotes problem" when giving it a final test too - now it works perfectly